libs/category/src/models/filters/type/range/request.ts

```diff
@@ -0,0 +1,3 @@
+import { DaffCategoryFilterRangeNumericRequest } from '../range-numeric/public_api';
+
+export type DaffCategoryFilterRangeRequest = DaffCategoryFilterRangeNumericRequest;
```

Comment:
I also think that setting something more generic to something more specific is a problem.

Reply:
There are some other concepts which make sense as ranges, e.g. dates or alphabetic characters. I know they can be represented under the hood as numbers but it should be cleaner to have specific typing.

Reply:
That's a good point. I still don't really understand this particular type though. 

Actually, I just realized that this isn't used anywhere. I think it can be deleted

Reply:
The idea is that when more types of ranges are added this type will be expanded to a union:

```ts
type DaffCategoryFilterRangeRequest = DaffCategoryFilterRangeNumericRequest | DaffCategoryFilterRangeDateRequest
```

Reply:
I agree this should be documented

Reply:
oh ok that makes sense. I'll leave this comment open till the documentation comes then

Reply:
@griest024 This got deleted. Will it just be readded if another range type filter is added later?

Reply:
We elected to delete it as the internal usage was actually incorrect typing. We may add it in the future, but currently, we don't have a use-case to support it.